### PR TITLE
Updated date format

### PR DESCRIPTION
**Pull request by amolswnz**

### CODE REVIEW

**Comment by Firesphere** [comment]
Please make the PR against the appropriate branch, for Meridian, a separate branch is made and PR's should always come from that branch

**Comment by amolswnz** [comment]
Updated the pull request. Please check.

File: code/Model/EditableFormField/EditableDateField.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function getCMSFields()
     public function getFormField()
     {
         $defaultValue = $this->DefaultToToday
-            ? DBDatetime::now()->Format('Y-m-d')
+            ? DBDatetime::now()->Format('y-MM-dd')
```
[the remark this answers is not in the document]
Comment (amolswnz) [reply code/Model/EditableFormField/EditableDateField.php]: Updated. Please check.

File: code/Model/EditableFormField/EditableDateField.php [hunk 2]
```diff
@@ -55,7 +55,7 @@ public function getCMSFields()
     public function getFormField()
     {
         $defaultValue = $this->DefaultToToday
-            ? DBDatetime::now()->Format('y-MM-dd')
+            ? DBDatetime::now()->Format(DBDatetime::ISO_DATE)
```
Comment (amolswnz) [inline code/Model/EditableFormField/EditableDateField.php]: Updated datetime format

**Pull request merged**